src/vs/workbench/services/search/common/textSearchManager.ts

```diff
@@ -29,60 +29,73 @@ export class TextSearchManager {
 	constructor(private query: ITextQuery, private provider: TextSearchProvider, private fileUtils: IFileUtils, private processType: ITextSearchStats['type']) { }
 
 	search(onProgress: (matches: IFileMatch[]) => void, token: CancellationToken): Promise<ISearchCompleteStats> {
-		const folderQueries = this.query.folderQueries || [];
 		const tokenSource = new CancellationTokenSource(token);
 
 		return new Promise<ISearchCompleteStats>((resolve, reject) => {
 			this.collector = new TextSearchResultsCollector(onProgress);
 
 			let isCanceled = false;
-			const onResult = (result: TextSearchResult, folderIdx: number) => {
+			const onResult = (result: TextSearchResult) => {
 				if (isCanceled) {
 					return;
 				}
 
 				if (!this.isLimitHit) {
 					const resultSize = this.resultSize(result);
-					if (extensionResultIsMatch(result) && typeof this.query.maxResults === 'number' && this.resultCount + resultSize > this.query.maxResults) {
+					if (
+						extensionResultIsMatch(result) &&
+						typeof this.query.maxResults === 'number' &&
+						this.resultCount + resultSize > this.query.maxResults
+					) {
```

Comment:
Can you undo the formatting changes? These will cause unnecessary conflicts when we upgrade to newer vscode versions and it also makes it harder to review.